src/main/resources/application.yaml

```diff
@@ -189,6 +190,7 @@ send-grid:
           SOA_PERSONAL_CA_DA_APPLICANT_LEGAL_REP: ${SEND_GRID_SOA_PERSONAL_CA_DA_APPLICANT_LEGAL_REP:d-13b61732cb744f89a9489d4b3500123a}
           SOA_SERVE_APPLICANT_SOLICITOR_NONPER_PER_CA_CB: ${SOA_SERVE_APPLICANT_SOLICITOR_NONPER_PER_CA_CB:d-8b6f127addd240b599528c75b3b37974}
           SOA_CA_LOCAL_AUTHORITY: ${SEND_GRID_SOA_CA_LOCAL_AUTHORITY:d-d121b680310f45e5a4117dd7023482e2}
+          SEND_EMAIL_TO_EXTERNAL_PARTY: d-d6f807d095b046458517924e6c48e5ea
   rpa:
```

Comment:
add overriding configuration constant to include template for production

Reply:
Addressed